backend-rust/src/migrations.rs

```diff
@@ -13,8 +13,8 @@ mod m0008_canonical_address_and_transaction_search_index;
 mod m0010_fill_capital_bound_and_leverage_bound;
 mod m0014_baker_metrics;
 mod m0015_pool_rewards;
-mod m0018_payday_stake_information;
-mod m0019_chain_update_events;
+mod m0019_payday_stake_information;
```

Comment:
You do not have to update the versions? Nothing is applied yet

Reply:
I had to introduce schema version in between representing the partial migration, so everything after 18 gets shifted.